src/main/java/Deadline.java

```diff
@@ -0,0 +1,13 @@
+public class Deadline extends Task{
+    protected String dueDate;
+
+    public Deadline(String task, String dueDate) {
+        super(task);
+        this.dueDate = dueDate;
+    }
+
+    @Override
+    public String toString() {
+        return "[D]" + super.toString() + " (by: " + dueDate + ")";
```

Comment:
`[D]`, `[E]` and `[T]` could potentially be replaced with a constant so that it can be easily changed in the future if needed.